types/html-minifier/v1/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Do we still need all these rules disabled in the v3 version, can you make a sweep and see if you can clean this up a little?

NOTE!... don't really care about v1, just v3!!